pkg/cli/image/info/info.go

```diff
@@ -132,6 +138,10 @@ func (o *InfoOptions) Run() error {
 			if len(src.Ref.Tag) == 0 && len(src.Ref.ID) == 0 {
 				return fmt.Errorf("--from must point to an image ID or image tag")
 			}
+			if !icspWarned && len(o.ICSPFile) > 0 && len(src.Ref.Tag) > 0 {
+				fmt.Fprintf(o.ErrOut, "warning: --icsp-file only applies to images referenced by digest and will be ignored for tags\n")
```

Comment:
Why it will be ignored for tags?
Our (assisted-service) users are using tags https://github.com/openshift/hive/blob/master/docs/using-hive.md#openshift-version